src/vimeoplayer/views/Vimeoplayer.js

```diff
@@ -0,0 +1,472 @@
+import $ from 'jquery';
+import _ from 'underscore';
+import Mediaplayer from 'picnic/mediaplayer/views/Mediaplayer';
+import ApiLoader from 'picnic/vimeoplayer/services/ApiLoader';
+
+var DATA_VIDEOID = 'vimeoid',
+	DEFAULTS = {
+		debug: false,
+		loader: new ApiLoader(),
+		trigger: 'a',
+		classLoading: 'loading',
+		classPlaying: 'playing',
+		playerHideSpeed: 300,
+		playerProgressSteps: 5,
+		playerProgressInterval: 1000,
+		playerOptions: {
+			autoplay: true
+		},
+	};
+
```

Comment:
Add namespace instead of constants to solve this issue: https://github.com/moccu/picnic/commit/f8862990cdc568eb26d1a476a775702d27c0a7a0